tdvt/tdvt/tdvt.py

```diff
@@ -435,27 +436,28 @@ def create_parser():
     subparsers = parser.add_subparsers(help='commands', dest='command')
 
     #Get information.
-    list_parser = subparsers.add_parser('list', help='list information', usage=list_usage_text)
+    list_parser = subparsers.add_parser('list', help='List information about tests and configurations.', usage=list_usage_text)
     list_group = list_parser.add_mutually_exclusive_group(required=True)
     list_group.add_argument('--ds', dest='list_ds', help='List datasource config.', required=False, default=None, const='', nargs='?')
     list_group.add_argument('--logical_config', dest='list_logical_configs', help='List available logical configs.', required=False, default=None, const='', nargs='?')
 
     #Actions.
-    action_group = subparsers.add_parser('action', help='setup stuff', usage=action_usage_text)
+    action_group = subparsers.add_parser('action', help='Various non-test actions.', usage=action_usage_text)
     action_group.add_argument('--setup', dest='setup', action='store_true', help='Create setup directory structure.', required=False)
     action_group.add_argument('--add_ds', dest='add_ds', help='Add a new datasource.', required=False)
     action_group.add_argument('--diff-test', '-dd', dest='diff', help='Diff the results of the given test (ie exprtests/standard/setup.calcs_data.txt) against the expected files. Can be used with the sql and tuple options.', required=False)
 
     #Run tests.
-    run_test_parser = subparsers.add_parser('run', help='run tests', parents=[run_test_common_parser], usage=run_usage_text)
-    run_test_parser.add_argument('-f', dest='run_file', help='Json file containing failed tests to run.', required=False)
+    run_test_parser = subparsers.add_parser('run', help='Run tests.', parents=[run_test_common_parser], usage=run_usage_text)
+    run_test_parser.add_argument('ds', help='Comma separated list of Datasource names or groups to test. See the \'list\' command.', nargs='+')
     run_test_parser.add_argument('--verify', dest='smoke_test', action='store_true', help='Verifies the connection to a data source against tests in your .ini file with SmokeTest = True.', required=False)  # noqa: E501
     run_test_parser.add_argument('--logical', '-q', dest='logical_only', help='Only run logical tests whose config file name matches the supplied string, or all if blank.', required=False, default=None, const='*', nargs='?')
     run_test_parser.add_argument('--expression', '-e', dest='expression_only', help='Only run expression tests whose config file name matches the suppled string, or all if blank.', required=False, default=None, const='*', nargs='?')
     
 
     #Run test pattern.
-    run_test_pattern_parser = subparsers.add_parser('run-pattern', help='run tests using a pattern', parents=[run_test_common_parser], usage=run_pattern_usage_text)
+    run_test_pattern_parser = subparsers.add_parser('run-pattern', help='Run single tests using a pattern.', parents=[run_test_common_parser], usage=run_pattern_usage_text)
```

Comment:
I don't want to seem nitpicky about wording, but would changing this to "Run **a** single test using a pattern." or "Run individual tests using a pattern." add clarity?

Reply:
good idea